setup.py

```diff
@@ -57,13 +57,19 @@ def get_long_description():
     license='BSD',
     packages=find_packages(exclude=['docs', 'tests']),
     ext_modules=[
-        Extension(str('zbarlight._zbarlight'), [str('zbarlight/_zbarlight.c')], libraries=['zbar']),
+        Extension(
+            str('zbarlight._zbarlight'),
+            [str('zbarlight/_zbarlight.c')],
+            extra_compile_args=['-std=c99'],
+            libraries=['zbar'],
+        ),
     ],
     include_package_data=True,
     zip_safe=False,
     setup_requires=[
         'setuptools',
     ],
+    install_requires=REQUIREMENTS,
     tests_require=REQUIREMENTS + [
```

Comment:
It has nothing to do with this PR, but `tests_require` is the list of *additional* packages to install for tests. There is no need to include `REQUIREMENTS` here.